tests/UI/pages/BO/orders/view.js

```diff
@@ -180,6 +182,53 @@ class Order extends BOBasePage {
   Methods
    */
 
+  // Methods for order actions
+  /**
+   * Is update status button disabled
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  isUpdateStatusButtonDisabled(page) {
+    return this.elementVisible(page, `${this.updateStatusButton},disabled`, 1000);
```

Comment:
```suggestion
    return this.elementVisible(page, `${this.updateStatusButton},disabled`, 1000);
```

Is it really a comma ?

Reply:
Yes it's a selector with comma

Reply:
I'm not sure with it, using comma means you try to find `<disabled>` element :thinking: 
ping @NeOMakinG

Reply:
```suggestion
    return this.elementVisible(page, `${this.updateStatusButton}[disabled]`, 1000);
```

Reply:
Thank you @Progi1984 , @PierreRambaud and @boubkerbribri :)
I will fix it!

Reply:
`.selector, disabled` means that it selects `.selector` elements AND `<disabled />` elements which aren't official elements, the `[disabled]` is what you were searching for 👍

Reply:
Done thx :+1: